cmd/loom/loom.go

```diff
@@ -648,10 +649,24 @@ func loadAppStore(cfg *config.Config, logger *loom.Logger, targetVersion int64)
 		if err != nil {
 			return nil, err
 		}
+		iavlStore, err = store.LoadMultiWriterAppStore(iavlStore, evmStore, db, cfg.AppStore.MaxVersions, cfg.AppStore.IAVLFlushInterval)
+		if err != nil {
+			return nil, err
+		}
+		_, evmVersion := evmStore.Version()
```

Comment:
Move all loading logic to `LoadMultiWriterAppStore`. `LoadMultiWriterAppStore` should return `MultiWriterAppStore` not `IAVLStore`